pyvista/__init__.py

```diff
@@ -9,23 +9,272 @@
 # Load default theme.  Must be loaded first.
 from pyvista._version import __version__
 from pyvista.plotting import *
-from pyvista.utilities import *
-from pyvista.core import *
+from pyvista.utilities import (
+    # arrays,
+    cells,
+    # common,
+    # errors,
+    # features,
+    fileio,
```

Comment:
This is here because we have a `pyvista.fileio.get_ext(...)` in `pyvista/plotting/plotting.py`, but we also have `pyvista.get_ext()` so we should probably remove either.